e2e/suites/list-views/file-libraries.test.ts

```diff
@@ -28,7 +28,7 @@ import { LoginPage, BrowsingPage } from '../../pages/pages';
 import { Utils } from '../../utilities/utils';
 import { RepoClient } from '../../utilities/repo-client/repo-client';
 
-describe('File Libraries', () => {
```

Comment:
this should remain unchanged. This is the main suite, that refers to File Libraries in general, and contains sub-suites for My Libraries and Favorite Libraries